Makefile

```diff
@@ -87,4 +87,7 @@ e2e:
 e2e-prod:
 	FOCUS="\[Real\]" ./hack/e2e.sh
 
+e2e-keyrotation:
+	FOCUS="\[KeyRotation\] \[Fake\]" TIMEOUT=70m ./hack/e2e.sh
```

Comment:
Remove the space to conform with the way we structure tags in the e2e make target (this is actually what we do everywhere else in openshift/kube).